distribution/lib/Standard/Base/0.0.0-dev/src/Random.enso

```diff
@@ -1,99 +1,307 @@
 import project.Any.Any
-import project.Data.Boolean.Boolean
-import project.Data.Json.JS_Object
-import project.Data.Numbers.Float
-import project.Data.Numbers.Integer
+import project.Data.Range.Range
 import project.Data.Text.Text
+import project.Data.Time.Date.Date
+import project.Data.Time.Date_Range.Date_Range
+import project.Data.Time.Time_Of_Day.Time_Of_Day
+import project.Data.Time.Time_Period.Time_Period
+import project.Data.Map.Map
+import project.Data.Set.Set
 import project.Data.Vector.Vector
 import project.Error.Error
 import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Nothing.Nothing
+import project.Runtime.Ref.Ref
 import project.System
+from project.Data.Boolean import Boolean, False, True
+from project.Data.Numbers import Float, Integer
+from project.Data.Range.Extensions import all
 
 polyglot java import java.lang.Integer as Java_Integer
 polyglot java import java.util.Random as Java_Random
 polyglot java import java.util.UUID
 polyglot java import org.enso.base.Random_Utils
 
-## PRIVATE
-   UNSTABLE
-   Returns a default seed to use for random number generation.
+type Random
+    ## PRIVATE
+       GROUP Random
```

Comment:
If it's `PRIVATE` then I'm not sure if there is sense in adding `GROUP`? groups are used for the CB but PRIVATE items are supposed to be hidden there.

Reply:
The plan is to expose this later, since end-users will definitely want to create instances. We just don't know how we want to control that yet. That's why this is there, and why there are tests for this class directly.

Reply:
Ok, I will not argue to remove it, although IMO if we will expose later then that will be the correct moment to think _how_ it will be exposed and add any kind of user facing stuff like `GROUP` tags.